### PR TITLE
LF-16747 - Mapbox v4 support, SSL for map tiles, mapbox examples

### PR DESCRIPTION
**Pull request by pcolombo**
https://livefyre.atlassian.net/browse/LF-16747 #review

### CODE REVIEW

**Comment by gobengo** [comment]
lgtm, @ross-pfahler  do you agree?

**Comment by ross-pfahler** [comment]
I like it.

File: src/views/map-view.js [hunk 1]
```diff
@@ -191,12 +192,21 @@ define([
...
+        var urlTemplate;
+
         this._map = new L.Map(this.el, this._leafletMapOptions).setView(
             this._leafletMapOptions.center || [0,0],
             this._leafletMapOptions.zoom || 2
         );
 
-        new L.TileLayer("http://{s}.tiles.mapbox.com/v3/"+this._mapboxTileOptions.mapId+"/{z}/{x}/{y}."+this._mapboxTileOptions.format)
+        // No Mapbox accessToken, use Mapbox v3
+        if(this._mapboxTileOptions.accessToken == null) {
```
Comment (ross-pfahler) [inline src/views/map-view.js]: space between ``if`` and ``(``

**Comment by gobengo** [comment]
@pcolombo  do you think the README should also be updated to explain the accessToken option?

**Comment by pcolombo** [comment]
@gobengo @ross-pfahler - Both addressed in most recent commit (9389474) #review.

**Comment by markdoten** [comment]
@gobengo @ross-pfahler @pcolombo I think this is ready to go whenever.

**Comment by markdoten** [comment]
Closing for rebase to RC: https://github.com/Livefyre/streamhub-map/pull/9

**Pull request closed without merging**